docs/docs-content/clusters/edge/cluster-management/agent-upgrade-airgap.md

```diff
@@ -0,0 +1,111 @@
+
+- An active Edge cluster deployed in airgap mode.
+
+- Linux Machine (Physical or VM) with an AMD64 architecture.
+
+- Palette API key. Refer to the [User Authentication](../../../user-management/authentication/api-key/create-api-key.md)
+  resource to learn how to create a Palette API key.
+
+- An Edge Native type cluster profile. Refer to
+  [Create Edge Native Cluster Profile](../site-deployment/model-profile.md) guide to learn how to create an Edge Native
+  cluster profile. You may also have other add-on profiles that you wish to attach to your cluster.
+
+- Content tags in your profiles highlight the exact location of container images to be downloaded.
+
+## Procedure
+
+### Identify the Target Agent Version
+
+1. The target agent version depends on the Palette instance you use to build content bundles for your Edge cluster. Use
+   the following request to identify the target version. If you are using self-hosted Palette or VerteX, replace
+   `api.spectrocloud.com` with the fully qualified domain name of your API endpoints. Replace the `apiKey` field with
+   your API key.
+
+   ```shell
+   curl --location 'https://api.spectrocloud.com/v1/services/stylus/version' \
+   --header 'apiKey: ******'
+   ```
+
+   You can expect a response similar to the following. In the following response, the target agent version is 4.5.5.
+
+   ```json {9}
+   {
+     "metadata": {
+       "creationTimestamp": "0001-01-01T00:00:00.000Z",
+       "deletionTimestamp": "0001-01-01T00:00:00.000Z",
+       "lastModifiedTimestamp": "0001-01-01T00:00:00.000Z"
+     },
+     "spec": {
+       "latestVersion": {
+         "content": "name: stylus\nversion: 4.5.5\nbuildId: \"20241025\"\nmajorVersion: \"4.5\"\napplyFilepath: /roar/stylus/4.5/4.5.5/apply/manifest.yaml\ndeleteFilepath: \"\"\n",
+         "name": "manifest.yaml",
+         "path": "nickfury/4.5/4.5.5/apply/services.yaml"
+       },
+       "name": "stylus"
+     }
+   }
+   ```
+
+### Upgrade Palette Agent Version
+
+2. Log in to [Palette](https://console.spectrocloud.com).
+
+3. From the left **Main Menu**, select **Profiles**. Select the profile you want your cluster to upgrade to.
+
+4. Create a new version of the profile. For more information, refer to
+   [Update a Cluster](../../cluster-management/cluster-updates.md).
+
+5. In the OS layer of the profile, include the following line. Replace `versionNumber` with your target agent version
+   number.
+
+   ```yaml {9}
+   pack:
+     content:
+       images:
+         - image: "{{.spectro.pack.edge-native-byoi.options.system.uri}}"
+
+   options:
+     system.uri: ttl.sh/ubuntu:k3s-1.29.5-v4.5.8-palette-demo
+
+   stylusPackage: container://us-docker.pkg.dev/palette-images/edge/stylus-linux-amd64:v<versionNumber>
+   ```
+
+   :::info
+
+   Following the EdgeForge process, it's likely that your provider image has a `peVersion` parameter. It is permissable
+   for this version to be different from the version you use for `stylusPackage`.
```

Comment:
```suggestion
   Following the EdgeForge process, it's likely that your provider image has a `peVersion` parameter. You can use a different version value in the `peVersion` parameter from the version you use for `stylusPackage`.
```

Let's avoid using "It is" and use more direct language instead. Using "It is" is probably one of my pet peeves in technical documentation 🙈